docs/source/integrations/plugins-event-reference.mdx

```diff
@@ -195,18 +222,24 @@ const server = new ApolloServer({
 })
 ```
 
+</MultiCodeBlock>
+
 If your plugin doesn't need to respond to any request lifecycle events, `requestDidStart`
 should not return a value.
 
 ### `schemaDidLoadOrUpdate`
 
-The `schemaDidLoadOrUpdate` event fires whenever Apollo Server initially loads the schema or updates the schema. A `schemaDidLoadOrUpdate` handler is given the new API schema and optionally the new core schema (if using a gateway). If you provide a gateway and it is older than `@apollo/gateway@0.35.0`, attempting to register a `schemaDidLoadOrUpdate` handler will fail.
+The `schemaDidLoadOrUpdate` event fires whenever Apollo Server initially loads the schema or updates the schema. 
+
+A `schemaDidLoadOrUpdate` handler is given the new API schema and optionally the new core schema (if using a gateway). If you provide a gateway and it is older than `@apollo/gateway@0.35.0`, attempting to register a `schemaDidLoadOrUpdate` handler will fail.
```

Comment:
Outdated gateway caveat here isn't applicable anymore since versions this old are unusable with AS4 (no graphql peer dependency overlap, mostly).

```suggestion
A `schemaDidLoadOrUpdate` handler is given the new API schema and optionally the new core schema (if using a gateway).
```